src/main/java/lotto/model/WinningPrice.java

```diff
@@ -0,0 +1,30 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+public enum WinningPrice {
+    NOTWIN(2, 0),
+    FORTH(3, 5_000),
+    THIRD(4, 50_000),
+    SECOND(5, 1_500_000),
+    FIRST(6, 2_000_000_000);
+
+    private final int winningCount;
+    private final int price;
+
+    WinningPrice(int winningCount, int price) {
+        this.winningCount = winningCount;
+        this.price = price;
+    }
+
+    public static WinningPrice of(int winningCount) {
+        return Arrays.stream(values()).filter(it -> it.winningCount == winningCount)
+                .findFirst()
+                .orElse(NOTWIN);
```

Comment:
개행과 축약하지 않은 변수명을 통해 가독성을 확보해 보면 어떨까 의견 드립니다.
```suggestion
        return Arrays.stream(values())
                .filter(winningPrice -> winningPrice.winningCount == winningCount)
                .findFirst()
                .orElse(NOTWIN);
```